core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala

```diff
@@ -62,14 +62,14 @@ class KryoSerializer(conf: SparkConf)
 
   if (bufferSizeKb >= ByteUnit.GiB.toKiB(2)) {
     throw new IllegalArgumentException("spark.kryoserializer.buffer must be less than " +
-      s"2048 mb, got: + ${ByteUnit.KiB.toMiB(bufferSizeKb)} mb.")
+      s"2048 mib, got: + ${ByteUnit.KiB.toMiB(bufferSizeKb)} mib.")
```

Comment:
Nit: mib -> MiB. (I know it was 'mb' before but that's not really the abbreviation for 1 million bytes)